client-hc/src/main/java/com/graphhopper/api/GraphHopperWeb.java

```diff
@@ -64,8 +64,8 @@
     private boolean calcPoints = true;
     private boolean elevation = false;
     private String optimize = "false";
-    boolean postRequest = true;
-    int maxUnzippedLength = 1000;
+    private boolean postRequest = true;
```

Comment:
This wasn't private before but should be now problem to do so as there is a setter anyway